docs/operational_guide/replication_between_clusters.md

```diff
@@ -0,0 +1,73 @@
+# Replication between clusters (beta)
+
+## Overview
+
+M3DB clusters can be configured to passively replicate data from other clusters. This feature is most commonly used when operators wish to run two (or more) regional clusters that function independently, but that passively replicate data from each other in an eventually consistent manner.
+
+The cross-cluster replication feature is built on-top of the [background repairs](repairs.md) feature. As a result, it has all the same caveats and limitations. Speifically, it does not currently work with clusters that use M3DB's indexing feature and the replication delay between two clusters will be at least (block size + bufferPast) for each namespace.
```

Comment:
nit: backticks and tokenization (`(block_size + buffer_past)`) - also, does this need to account for `block_start` as the earlier example did?

Reply:
it doesn't need to account for `block_start` because this is worst-case replication delay but I'll fix the backtick stuff

Reply:
actually I see what you're asking. Updated it slightly to make it more clear